Dockerfiles/Dockerfile.dela

```diff
@@ -2,12 +2,16 @@ FROM golang:1.20.6-bookworm
 
 RUN apt-get update && apt-get install git
 RUN git clone https://github.com/dedis/dela.git
-RUN git clone https://github.com/dedis/d-voting.git
+WORKDIR /go/dela
+# make sure we're using the same head as d-voting
+RUN git clone -b fix-bbolt
```

Comment:
is this a specific fix for d-voting ? or is it something that should be integrated in dela ?

Reply:
there is a PR in Dela (https://github.com/dedis/dela/pull/270), but it's not merged yet as there are some on-going questions (for Dela) - but d-voting doesn't work correctly without this fix, so d-voting points to this branch at the moment, see https://github.com/dedis/d-voting/pull/347

the changes in the Dockerfile just make sure that the same version of DELA is used as for d-voting